chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -181,11 +181,15 @@ const initStartChat = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAct
             dispatch({ type: LiveChatWidgetActionType.SET_START_CHAT_FAILING, payload: false });
             dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.Active });
         }
-
+        const isPostchatEnabled = (): boolean => {
```

Comment:
Can we have a new method in `common` to check for the post-chat survey flag?

You can technically use `chatSDK` to retrieve the `chatConfig` via `chatSDK.getLiveChatConfig()`. At this point of time, chatSDK would have been initialized already, chatSDK would already have the chatConfig value at runtime so it would retrieve it from there instead of making an additional API call.

```
const chatConfig = chatSDK.getLiveChatConfig();
return !!(chatConfig?.LiveWSAndLiveChatEngJoin?.msdyn_postconversationsurveyenable 
                   ?? state?.domainStates?.liveChatConfig?.LiveWSAndLiveChatEngJoin?.msdyn_postconversationsurveyenable)
```

Reply:
Created a new method called livechatConfigUtils.ts